common.js

```diff
@@ -68,6 +68,11 @@ function setopts (self, pattern, options) {
   self.nodir = !!options.nodir
   if (self.nodir)
     self.mark = true
+  self.supportWindowsPaths = !!options.supportWindowsPaths
+  if (self.supportWindowsPaths) {
+    options.pathSep = undefined
+    pattern = pattern.replace(/\\/g, "/")
```

Comment:
Cool things I found out, `/\\/g` is **105% faster** than `/\\|\//g`.

Reply:
So instead of doing 1000/ms you can do 2000?  Not trying to be too snarky, but it's a one-time action at initialization, and wildly fast, even in a benchmark with 2000 path portions and a mix of `\` and `/`.  Even if it was 10,000x faster, even if it was somehow able to magically do it in 0 CPU cycles, it's still only saving you 0.0001% of the overall time.

Reply:
Of course, if it was actually possible to do in 0 CPU cycles, we'd be better off spending our time figuring how how to express all compute problems in terms of slash replacement, and win the turing prize for solving P=NP lol

Reply:
I know that it's a very minor improvement in the grand scheme of things, it just seemed pretty cool to me.